### DISCUSSION TITLE
Sample policy disallow-capabilities-strict might have lines 79 and 81 swapped

### DISCUSSION THREAD

**Discussion by cybra-northco** [kyverno/policies#1031] [Q&A] [answered]
At a first glance, it seems like lines 79 and 81 should be swapped in https://github.com/kyverno/kyverno/blob/main/charts/kyverno-policies/templates/restricted/disallow-capabilities-strict.yaml

Or do I misunderstand the validation rule?

**Comment by chipzoller** [comment, marked as answer]
No, this is correct. If you swapped them it wouldn't work correctly in all scenarios.
